service/src/main/java/com/codeforcommunity/processor/MapProcessorImpl.java

```diff
@@ -293,7 +294,7 @@ public SiteGeoResponse getSiteGeoJson() {
   private static class SiteGeoResponseCache {
     private static SiteGeoResponse response;
     private static long expireTime = 0;
-    private static final long timeToLive = 10000; // in milliseconds, 5000 is the avg. response time
+    private static final long timeToLive = 1 * 60 * 1000; // in milliseconds, 5000 is the avg. response time
```

Comment:
Upping the cache time to 1 minute - we probably could do more, but it also means deleting/adding sites won't be visible until after the cache expires

Reply:
this is to cache the data for the home page map? What is the reasoning behind upping cache time to minute?

Reply:
is since filtering takes extra query time and so less frequent cache rates will not make it so expensive

Reply:
Yeah, since this method requires a lot of work to query + process the data, the idea here is that increasing the cache time will reduce the load on our database/backend. Since it's just a super simple cache though, it'll mean any updates to the map may take up to a minute to show up for the users - I think I'm gonna dial this back just a bit so the UX doesn't suffer too much, and if it does get really bad we can start re-thinking how we cache the response.